Makefile

```diff
@@ -187,10 +187,16 @@ clean-target-dir:
 
 # Dependencies
 
-.venv/bin/python: Makefile
+.venv/bin/python:
 	@rm -rf .venv
-	@which virtualenv || sudo pip install virtualenv
-	virtualenv -p $$RELAY_PYTHON_VERSION .venv
+ifeq ($(TRAVIS)$(basename $(RELAY_PYTHON_VERSION)), python3)
+	@# if (!TRAVIS && python3)
+	$(RELAY_PYTHON_VERSION) -m venv .venv
+else
+	@# if (TRAVIS || python2)
+	@# force use virtualenv in TRAVIS because of https://github.com/travis-ci/travis-ci/issues/8589
+	virtualenv -p $(RELAY_PYTHON_VERSION) .venv
+endif
```

Comment:
This got ugly 😞 

Alternatively, we don't need to create a virtualenv in Travis, because it already does create one for us.
I opted for not doing that right now though, as it requires a lot more changes to places where we run `./venv/bin/FOO`.

As is it reduces dev requirements.

Reply:
TBH, this is worse than what we had before. If `virtualenv` is an issue, we can just document that this is required - as for many other sentry projects like `sentry`, `symbolic`, `symbolicator`, `snuba` and so on.